plugin-server/package.json

```diff
@@ -69,16 +69,17 @@
         "express": "^4.18.2",
         "faker": "^5.5.3",
         "fast-deep-equal": "^3.1.3",
+        "fastpriorityqueue": "^0.7.5",
```

Comment:
Maybe rebase on master is needed?

Reply:
My previous PR added it to the main package.json, not the plugin-server one